### PR TITLE
runtime(zip-plugin): Fix extract file from zip browser

### PR DESCRIPTION
**Pull request by ghost**
Problem:  Enter 'x' in zip browser fail with E121
Solution: Fix typo in zip#Extract()

### CODE REVIEW

File: runtime/autoload/zip.vim [hunk 1]
```diff
@@ -395,8 +395,8 @@ fun! zip#Extract()
   endif
 
   " extract the file mentioned under the cursor
-"  call Decho("system(".g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(shell).")")
-  call system(g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(shell))
+"  call Decho("system(".g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(fname).")")
+  call system($"{g:zip_extractcmd} {shellescape(b:zipfile)} {shellescape(fname)}")
```
Comment (zzzyxwvut) [inline runtime/autoload/zip.vim:399]: The support for string interpolation is fairly recent
`v8.2.4883` (May 2022).  Substituting `fname` should be enough?

Comment (ghost) [reply runtime/autoload/zip.vim:399]: As this plugin is now maintained by the Vim project, I thought there was no problem.
Any opinions?

Comment (nickspoons) [reply runtime/autoload/zip.vim:399]: I thought the same, it is more readable to my eye with the string interpolation

Comment (chrisbra) [reply runtime/autoload/zip.vim:399]: should be fine to use string interpolation. We do not make any guarantees, that our runtime files need to be executable for older Vims anyhow. I would also just remove the commented out `Decho` call.

**Comment by chrisbra** [comment]
thanks!

**Pull request closed without merging**